packages/native/src/components/MapView/MapCallout.native.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+import { Callout } from "react-native-maps";
+import { CalloutProps } from "./types";
+import {
+  COMPONENT_TYPES,
+  createActionProp,
+  createBoolProp,
+} from "@draftbit/types";
+
+const MapCallout: React.FC<CalloutProps> = ({
+  onPress,
+  showTooltip,
+  children,
+}) => (
+  <Callout tooltip={showTooltip} onPress={onPress}>
```

Comment:
Passing true for showTooltip will actually not show bubble around. 
I would use it like `<Callout tooltip={!showBubble} ...>`